helix-rest/pom.xml

```diff
@@ -44,6 +44,11 @@ under the License.
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.helix</groupId>
+      <artifactId>metadata-store-directory-common</artifactId>
+      <version>0.9.2-SNAPSHOT</version>
```

Comment:
I think it should be 0.9.5-SNAPSHOT? We already have 0.9.4 release. We need change all the places for the pom.

Reply:
Last time I asked, the answer is that we have not released 0.9.2 officially yet.

Reply:
0.9.2 throughout zooscalability branch. 0.9.4 is not an official release, so we'll have to do some manual updating.